documentation-tree/react/mocking/stripe.md

```diff
@@ -0,0 +1,104 @@
+# Mocking Stripe
+
+## Component
+
+We have this component, which roughly looks like this:
+
+```
```

Comment:
You can add syntax highlighting with: '''js

Reply:
Thanks @Braden1996 will do!

Reply:
That's such a cooool tip!